gapic/templates/%namespace/%name_%version/%sub/types/_message.py.j2

```diff
@@ -75,4 +75,22 @@ class {{ message.name }}({{ p }}.Message):
     )
     {% endif %}{# field.map #}
     {% endfor %}{# for field in message.fields.values#}
+
+    {% if message.extended_operation_status_field %}
+    {% with status_field = message.extended_operation_status_field %}
+    @property
+    def done(self) -> bool:
+        """Return True if the backing extended operation is completed, False otherwise."""
+        {% if status_field.enum %}
+        return self.{{ status_field.name }}.name == "DONE"
```

Comment:
is this .name intended?

Reply:
Yes it is true. To me it seems the second variant that Dov provided is better, as it uses enums directly, but I'm fine with either one.